### PR TITLE
Say that -webkit-appearance must be supported

### PR DESCRIPTION
**Pull request by zcorpan**
Also say that appearance must not be supported.

### CODE REVIEW

File: compatibility.bs [hunk 1]
```diff
@@ -502,6 +502,12 @@ The following <code>-webkit-</code> <a>vendor prefixed</a> properties must be su
 <br>
 <div class="note">As soon as each property is unprefixable it can be defined as a "simple" alias.</div>
 
+<h4 id="css-non-alias">Non-alias vendor prefixed properties</h4>
```
Comment (miketaylr) [inline compatibility.bs]: I think "non-aliased" reads as more natural, at least to me. But I could be convinced either way.

(unless non-alias is the more commonly used technical term i'm not familiar with)

Comment (miketaylr) [reply compatibility.bs]: (feel free to change and merge after -- or just merge if you think `non-alias` is better here)

**Comment by miketaylr** [comment]
I guess we want to have some wpt as well -- I thought we did for `-webkit-appearance`, but I see that was backed out: https://github.com/web-platform-tests/wpt/commit/4981a13ac22db892707dcaac3a6678e623224268#diff-6805d2eb09708dde50e925b430024691

**Comment by zcorpan** [comment]
PTAL

**Pull request merged**